doc/README.org

```diff
@@ -25,7 +25,18 @@ cat ../version
 - [[https://www.quicklisp.org/beta/][Quicklisp]]
 - curl (refer to your system package manager)
   
-** INSTALLATION
+** AUTOMATIC INSTALLATION
+The default implementation is sbcl, if you are using another then set the ~LANG~
+environment variable, for example to use Clozure common lisp:
+#+begin_src lisp
+export LANG=ccl
+#+end_src
+now run the installer script:
+#+begin_src sh
+curl https://raw.githubusercontent.com/rudolfochrist/ql-https/master/install.sh | bash
```

Comment:
Wouldn't it be better, instead of exporting LANG, to set LANG and run `curl` in one statement. Like 
```
LANG=ccl curl https://raw.githubusercontent.com/rudolfochrist/ql-https/master/install.sh | bash
```

Reply:
It would be, but the problem is that just sets it for curl and not while the actual bash script runs, it'd need to be:
```
curl https://raw.githubusercontent.com/rudolfochrist/ql-https/master/install.sh | LANG=ccl bash
```
and I thought with the LANG= in the middle of the line there it might not be so clear, but I guess it'd be ok.
Another option that might be better would be not using `curl | bash` and have the instructions be like:
```
git clone https://github.com/rudolfochrist/ql-https ~/common-lisp/ql-https
LANG=ccl ~/common-lisp/ql-https/install.sh
```

Reply:
oops I keep mixing up LANG and LISP lol

Reply:
Ah yes. Totally missed the pipe there. But since it's actually `LISP` and not `LANG` I'm also fine with exporting it.